.github/workflows/build.yml

```diff
@@ -93,15 +93,27 @@ jobs:
 
   tests:
     needs: build
-    name: ${{ matrix.grunt-cmd }}
+    name: ${{ matrix.grunt-cmd }}_${{ matrix.suite ||'' }}
```

Comment:
Can you please change the separator from `_` for `-` (across all the PR)?
I'd like to have some consistency in the names of the jobs, because now they look like this: `ci-webdriver-default_basic`